Use the options_text as the variant title

thisiscab opened the pull request:
That way it's easier to recognize what each variants are. Right now we
were using the SKUs number which was not very self explanatory.

https://app.clubhouse.io/dynamomtl/story/108/as-a-pos-user-i-want-to-see-the-variant-title-instead-of-the-variant-skus

thisiscab committed:
Use the options_text as the variant title

That way it's easier to recognize what each variants are. Right now we
were using the SKUs number which was not very self explanatory.

https://app.clubhouse.io/dynamomtl/story/108/as-a-pos-user-i-want-to-see-the-variant-title-instead-of-the-variant-skus

thisiscab commented:
Hey buddies! Now is your time to shine!

@braidn @DanielWright

thisiscab committed:
Will stop rubocop from complaining

DanielWright reviewed (approved):
Looks good to me. I like how easy it was to change this, good-design points. 😄

thisiscab committed:
Change the variant uniqueness constraint instead of title

We can't manually change the title. Shopify takes the first option_value
of the variant take that has the title of the variant.
